hpe-visualizations-1.0.sketch

JetLim opened the pull request:
REFINED 
- Reduced Symbols
- Refined ‘value’ alignments for Legend elements
- Updated ‘Marker’ line color to solid Hex#2AD2C9 based on HPE set of colors.
- Symbolized ‘Limits’ element

JetLim committed:
hpe-visualizations-1.0.sketch

REFINED 
- Reduced Symbols
- Refined ‘value’ alignments for Legend elements
- Updated ‘Marker’ line color to solid Hex#2AD2C9 based on HPE set of colors.
- Symbolized ‘Limits’ element

JetLim committed:
hpe-visualizations-1.0.sketch

- Turned off 'Grid' for Meter Series, Meter Value.

L0ZZI commented:
Changes:

- Distribution, "Icon" type, make the icons respond proportionally? Right now they are stretching.
- Chart, "Meter", "Meter Value" adjust baseline of bars to the bottom of the grid (no overhang).
- Meter, BarSeries and other places have the Labels truncating on resize. Need to the change your text box property from "fixed" to "auto"
-

JetLim committed:
Fixed Distribution Chart and Meter feedbacks.

Fixed - Distribution, "Icon" type, make the icons respond proportionally? Right now they are stretching.
Fixed - Chart, "Meter", "Meter Value" adjust baseline of bars to the bottom of the grid (no overhang).
Fixed - Meter, BarSeries and other places have the Labels truncating on resize. Need to the change your text box property from "fixed" to "auto"

L0ZZI commented:
Notes for the next version: 

- Responsive Groups on Circle Meter Value, Value is stretching apart 
- Responsiveness of Value on Circle Meter Series is getting swapped…
- Pinning of Value on Bar Meter is not Pinned
- Chart Line dots are stretching on respond, should stay anchored and proportional
- Chart Bar dots are stretching on respond, should stay anchored and proportional
- Chart Meter Series increments responding in groups, not floating
- Chart Meter Value increments responding in groups, not floating
